feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.html

```diff
@@ -15,14 +15,6 @@
       (keydown.enter)="bindCartToCustomer()"
       [attr.aria-label]="'asm.bindCart.enterCartId' | cxTranslate"
     />
-    <button
-      class="cx-asm-reset"
-      [attr.aria-label]="'asm.bindCart.resetCartId' | cxTranslate"
-      [class.visible]="cartId.value?.length > 0"
-      (click)="clearText()"
```

Comment:
clearText() function in ts file also should delete

Reply:
https://github.com/SAP/spartacus/pull/19078/files#diff-06efd864d25531358f2a301a49ff68ed3ffc5949b6b39eb5083ebf8ce72a1bf9L150
also should delete

Reply:
have been modified.